redis/src/aio.rs

```diff
@@ -572,18 +605,28 @@ where
             }
 
             let mut rv = Vec::with_capacity(count);
-            for _ in 0..count {
+            let mut count = count;
```

Comment:
Is it ever possible we'd need the same push-command handling logic on `req_packed_command`?

Reply:
Yep and now it's added to both aio & sync connections also tests been updated.